server-phoenix/config/dev.exs

```diff
@@ -2,6 +2,8 @@ import Config
 
 # Configure your database
 
+# change this to postgres -> sqlite3 temporarily (ecto setup)
+# iex -S mix (starts the phoenix)
```

Comment:
We can probably drop these comments for a note in the commit message about sqlite3 being a temporary solution